browser/ui/webui/playlist_page_handler.cc

```diff
@@ -72,6 +76,26 @@ void PlaylistPageHandler::AddMediaFilesFromPageToPlaylist(const std::string& id,
                                                                   url.spec());
 }
 
+void PlaylistPageHandler::AddMediaFilesFromOpenTabsToPlaylist(
+    const std::string& playlist_id) {
+  std::vector<content::WebContents*> contentses;
+
+  auto* browser = chrome::FindBrowserWithWebContents(web_contents_);
+  DCHECK(browser);
+  auto* tab_strip_model = browser->tab_strip_model();
+  for (auto i = 0; i < tab_strip_model->count(); i++) {
+    if (auto* contents = tab_strip_model->GetWebContentsAt(i);
+        contents != web_contents_) {
```

Comment:
IMO, this checking could be improved. Instead of comparing web contents itself, how about checking whether it loaded supported service (ex, youtube.com) or not?
Crash happened when this api is called when there are no other tabs that loaded youtube.com.

Reply:
> IMO, this checking could be improved. Instead of comparing web contents itself, how about checking whether it loaded supported service (ex, youtube.com) or not?

Oops, this isn't a thing to filter service. This is temporal code to avoid targeting `brave://playlist` itself. The crash seems to be from CHECK(). That should be handled in other way.

Reply:
Thank you for letting me know the crash. It should be gone now!